src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -2170,10 +2126,16 @@ public void copyVoltageInitModifications(UUID studyUuid, UUID nodeUuid, String u
     }
 
     @Transactional
```

Comment:
Add readOnly ?

Reply:
Nope because it can create.
https://github.com/gridsuite/study-server/blob/40163fa22228c4d51bc6ffdc85f9ab430316f37f/src/main/java/org/gridsuite/study/server/service/SensitivityAnalysisService.java#L275